proto/api/v1/cluster_service.proto

```diff
@@ -23,8 +23,19 @@ enum LoadBalancerType {
     ROUTE         = 3;
 }
 
+// next available tag: 3
+message DecommissionedClusterRetentionInfo {
+    oneof RetentionInfo {
+        // indicates whether a cluster is protected from deletion
+        bool is_protected_cluster    = 1;
```

Comment:
naming suggestion earlier

Reply:
UI code convention strongly prefers `is` or `has` prefix for boolean state values, so also prefer something like `is_excluded_from_decommissioning` which becomes `isExcludedFromDecommissioning` in JSON